processing/heartbeat_injector.go

```diff
@@ -0,0 +1,127 @@
+package processing
+
+// DCSO FEVER
+// Copyright (c) 2020, DCSO GmbH
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"regexp"
+	"time"
+
+	"github.com/DCSO/fever/types"
+	"github.com/DCSO/fever/util"
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	// match 24 hour local time string, separated by colon
+	injectTimeRegex     = regexp.MustCompile(`^(([01][0-9])|(2[0-3])):[0-5][0-9]$`)
+	injectTimeCheckTick = 40 * time.Second
```

Comment:
Is this hard coded time a default value that wants configuration? If not why is it 40 second? Maybe "found by experience" would be a comment that is enough otherwise.

Reply:
That was me trying to avoid missing one of the specified times when the check takes part at a minute boundary -- at the cost of potentially triggering a heartbeat more than once on a specified minute, which would be fine for me.
But I guess that is probably too much and is more understandable if we just pick 1 minute as the tick interval, it is just more intuitive.

Reply:
Hardcoded value changed and comment added.